### ISSUE TITLE
can you public InstantAdapterCore?

### ISSUE THREAD

**Issue by jiqimaogou** [ragunathjawahar/adapter-kit#9]
can you public InstantAdapterCore class? so I can use it's bind method.

**Comment by ragunathjawahar** [comment]
@jiqimaogou For now can you clone the project and use it? [InstantAdapterCore](https://github.com/mobsandgeeks/adapter-kit/blob/master/library/src/com/mobsandgeeks/adapters/InstantAdapterCore.java).

**Comment by jiqimaogou** [comment]
I think these can use apt to generate somecode, not use reflection which is low performance..

**Comment by ragunathjawahar** [comment]
@jiqimaogou exactly, that's why there hasn't been any development in this project lately.

**Comment by jiqimaogou** [comment]
I think use apt, this repo would be a great repo.which save many times.

**Comment by ragunathjawahar** [comment]
@jiqimaogou I rewrote the same project using APT a year ago, but it was more of a PoC. I have similar plans (we have the all new `RecyclerView`) but don't have a timeline on this yet.
